ncc & v3.1.0

Hello,

I'm using [ncc](https://github.com/vercel/ncc) to bundle a vscode extension which is using this library.
With `v3.0.0`, I don't have any issue.
With `v3.1.0`, `ncc` doesn't produce any errors but when running the extension, I get `Cannot find module './format'`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,12 @@
+
+3.2.0 2022-08-30
+=================
+- update the version of the bundled Javascript files to `es2020`.
+- include all `const enum` values in the bundled JavaScript files (`ScanError`, `SyntaxKind`, `ParseErrorCode`).
+
 3.1.0 2022-07-07
 ==================
-  * added new API `FormattingOptions.keepLines` : It leaves the initial line positions in the formatting
+  * added new API `FormattingOptions.keepLines` : It leaves the initial line positions in the formatting.
 
 3.0.0 2020-11-13
 ==================

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "jsonc-parser",
-  "version": "3.1.0",
+  "version": "3.2.0",
   "description": "Scanner and parser for JSON with comments.",
   "main": "./lib/umd/main.js",
   "typings": "./lib/umd/main.d.ts",
@@ -16,12 +16,12 @@
   },
   "devDependencies": {
     "mocha": "^10.0.0",
-    "typescript": "^4.7.4",
+    "typescript": "^4.8.2",
     "@types/node": "^16.x",
     "@types/mocha": "^9.1.1",
-    "@typescript-eslint/eslint-plugin": "^5.30.5",
-    "@typescript-eslint/parser": "^5.30.5",
-    "eslint": "^8.19.0",
+    "@typescript-eslint/eslint-plugin": "^5.36.0",
+    "@typescript-eslint/parser": "^5.36.0",
+    "eslint": "^8.23.0",
     "rimraf": "^3.0.2"
   },
   "scripts": {

--- a/src/tsconfig.esm.json
+++ b/src/tsconfig.esm.json
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "ES5",
+		"target": "es2020",
 		"module": "es6",
 		"moduleResolution": "node",
 		"sourceMap": true,
@@ -8,9 +8,9 @@
 		"stripInternal": true,
 		"outDir": "../lib/esm",
 		"strict": true,
+		"preserveConstEnums": true,
 		"lib": [
-			"es5",
-			"es2015.promise"
+			"es2020"
 		]
 	}
 }

--- a/src/tsconfig.json
+++ b/src/tsconfig.json
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "ES5",
+		"target": "es2020",
 		"module": "umd",
 		"moduleResolution": "node",
 		"sourceMap": true,
@@ -8,9 +8,9 @@
 		"stripInternal": true,
 		"outDir": "../lib/umd",
 		"strict": true,
+		"preserveConstEnums": true,
 		"lib": [
-			"es5",
-			"es2015.promise"
+			"es2020"
 		]
 	}
 }

